src/routes/users.js

```diff
@@ -36,7 +36,7 @@ module.exports = (() => {
 	*          "message": "User not found"
 	*     }
 	*/
-	router.get('/:email', userController.getUserIdByEmail);
+	router.get('/:email', userController.getUserByEmail);
```

Comment:
you need to update the success response docs here right? since it now returns more than the id

Reply:
thank you! i knew i was forgetting something.